exonum-java-binding/CHANGELOG.md

```diff
@@ -18,6 +18,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 The release is based on Exonum 0.11
 
 ### Added
+- Support of packaging the Exonum Java application into a single archive with all the necessary
+  dependencies. For the instructions, consult [CONTRIBUTING.md](../CONTRIBUTING.md#Building-Exonum-Java-App).
```

Comment:
I think those are the wrong instructions — why would you bother building (as previously) when a binary is provided? I think the first thing that must be provided (linked) is the installation instructions, and only then — how to build in case you need/want to continue doing so.